primer/src/Primer/Action/Available.hs

```diff
@@ -167,7 +167,8 @@ data NoInputAction
   | DeleteTypeParam
   | MakeKType
   | MakeKFun
-  | DeleteKind
+  | -- TODO: RaiseKind?
```

Comment:
I assume you meant to leave a review comment here?

Anyway, yes, we probably should have a "raise kind" action. Though I don't mind whether that's in a separate PR.

Reply:
See also https://github.com/hackworthltd/primer/issues/1141#issue-1892600830.

Reply:
This is now tracked in #1145